lib/ansible/modules/windows/win_find.ps1

```diff
@@ -250,23 +250,28 @@ Function Get-FileStat($file) {
         try {
             $lnk_source = [Ansible.Command.SymLinkHelper]::GetSymbolicLinkTarget($file)
             $file_stat.lnk_source = $lnk_source
-        } catch {}
+        } catch {
+	    Add-Warning -obj $result -message "symlink source retrieval error: $($_.Exception.Message)"
```

Comment:
I would hold off on the warnings here for now. There's a GitHub issue that's in place to fix up the error handling for win_find and this would be part of a bigger change to handle stuff like this.